spec/services/random_direct_reports_generator_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe RandomDirectReportsGenerator, type: :model do
+  let(:supervisor) { FactoryBot.create :staff_profile }
+
+  before do
+    supervisor # call this here so we do not create the supervisor as part of the test
+  end
+
+  describe "##create_reports" do
```

Comment:
what does the `##` mean?

Reply:
I believe ## means class method.

Reply:
ah, okay. I usually see `.` for class method? but 👍 this works now that I know 😄

Reply:
I'm not sure what convention I was following, but I am happy to follow the local convention.  I changed them to `.`.